configuration.rst

```diff
@@ -1,3 +1,7 @@
+.. meta::
+    :description: Customise and configure Gauge using the gauge.properties and the default.properties files.
```

Comment:
Customise and configure Gauge using gauge.properties and default.properties files.